tests/suites/test_suite_ssl.function

```diff
@@ -908,6 +908,34 @@ int mbedtls_move_handshake_to_state( mbedtls_ssl_context *ssl,
         }                                       \
     } while( 0 )
 
+void set_ciphersuite( mbedtls_ssl_config *conf, const char *cipher,
+                      int* forced_ciphersuite )
+{
+    const mbedtls_ssl_ciphersuite_t *ciphersuite_info;
+    forced_ciphersuite[0] = mbedtls_ssl_get_ciphersuite_id( cipher );
+    forced_ciphersuite[1] = 0;
+
+    ciphersuite_info =
+        mbedtls_ssl_ciphersuite_from_id( forced_ciphersuite[0] );
+
+    TEST_ASSERT( ciphersuite_info->min_minor_ver <= conf->max_minor_ver );
```

Comment:
Might we dereference `NULL` if `ciphersuite_info` is invalid?

Reply:
Yes we can, unfortunately. Adding a check for it.